frontends/web/src/locales/en/app.json

```diff
@@ -1655,6 +1664,20 @@
       "up-to-date": "Your app is up to date",
       "version": "App Version"
     },
+    "notes": {
+      "export": {
+        "description": "Create a backup of your transaction notes and account names.",
+        "success": "Notes successfully exported.",
+        "title": "Export notes"
+      },
+      "import": {
+        "description": "Restore your transaction notes and account names from a previously made backup file.",
+        "success": "Imported {{accountNamesLabel}}\nand {{transactionNotesLabel}} from {{filename}}",
```

Comment:
In my experience it is not a good idea to have chunks of a sentence translated individually, as it may not work well in all languages.

At this point it's probably better to formulate it in a way where we don't depend on multiple pluralizations per sentence, like:

```
"successAccounts_one": "Imported {{count}} account name",
"successTransactions_one": "Imported {{count}} transaction note",
...
```
and just have two sentences in the dialog?

Reply:
current version: 

<img width="699" alt="Screenshot 2024-06-10 at 15 49 38" src="https://github.com/benma/bitbox-wallet-app/assets/546900/f89b6b75-2104-4fc2-b38e-5ced1dfdd6be">
<img width="709" alt="Screenshot 2024-06-10 at 15 48 58" src="https://github.com/benma/bitbox-wallet-app/assets/546900/6e8b49d7-7846-42f0-a02f-94ccc5c1597d">

cc @benma @jadzeidan